library/arangodb

```diff
@@ -10,6 +10,6 @@ Tags: 3.8, 3.8.6
 GitCommit: af354ba85e29e2bee525677b45a321530235a38d
 Directory: alpine/3.8.6
 
-Tags: 3.9, 3.9.0, latest
-GitCommit: de7e9211691614f105c7809cb13405ef5afa0be8
-Directory: alpine/3.9.0
+Tags: 3.9, 3.9.1 latest
```

Comment:
It looks like the comma got lost :see_no_evil:

Reply:
Thanks for the catch!